src/db_ledger.rs

```diff
@@ -281,6 +281,7 @@ pub struct DbLedger {
     meta_cf: MetaCf,
     data_cf: DataCf,
     erasure_cf: ErasureCf,
+    pub new_blobs_signal: (Condvar, Mutex<bool>),
```

Comment:
why pub?

Reply:
It's being consumed here: https://github.com/solana-labs/solana/pull/2597/files#diff-3d5b06a60c5839fe958a3e8438bce845R306, I could provide public accessors to each of the elements in the tuple if we feel that's better encapsulation

Reply:
how about calling db_ledger.wait()?